<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>book.png</normaloff>book.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="edtText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cbxCatalogs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxBooks">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRead">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/read.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxLan"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnFont">
        <property name="toolTip">
         <string extracomment="Шрифт"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/font.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPattern">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>Глава</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>Раздел</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="lstChapters">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>101</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="lstSections">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>101</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAuthorization"/>
   <addaction name="actionCatalogsEditor"/>
   <addaction name="actionNotes"/>
   <addaction name="actionContent"/>
   <addaction name="actionSearch"/>
   <addaction name="actionExport"/>
   <addaction name="actionInfoDialog"/>
  </widget>
  <action name="actionNotes">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Заметки/Осознания</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/book.png</normaloff>:/images/book.png</iconset>
   </property>
   <property name="text">
    <string>Содержание книги</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionInfoDialog">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/information.png</normaloff>:/images/information.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionCatalogsEditor">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/catalogs-editor.png</normaloff>:/images/catalogs-editor.png</iconset>
   </property>
   <property name="text">
    <string>Редактирование каталогов</string>
   </property>
  </action>
  <action name="actionAuthorization">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/login-password.png</normaloff>:/images/login-password.png</iconset>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
   <property name="toolTip">
    <string>Авторизация</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/read.png</normaloff>:/images/read.png</iconset>
   </property>
   <property name="text">
    <string>Читать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../bin/images.qrc"/>
 </resources>
 <connections/>
</ui>
